responder/__init__.py

```diff
@@ -1,9 +1,20 @@
+from importlib.metadata import PackageNotFoundError, version
+
 from . import ext
 from .core import API, Request, Response
 
+__appname__ = "responder"
```

Comment:
I've never seen `__appname__` before. What's this from?

Reply:
Probably just a stray pickup. Let's remove it again.

Reply:
Removed.